packages/components/src/ui/color-picker/styles.ts

```diff
@@ -6,13 +6,37 @@ import styled from '@emotion/styled';
 /**
  * Internal dependencies
  */
+import NumberControl from '../../number-control';
 import InnerSelectControl from '../../select-control';
 import InnerRangeControl from '../../range-control';
 import { StyledField } from '../../base-control/styles/base-control-styles';
+import { ThumbWrapper } from '../../range-control/styles/range-control-styles'
 import { space } from '../utils/space';
+import {
+	BackdropUI,
+	Container as InputControlContainer,
+} from '../../input-control/styles/input-control-styles';
+import InputControl from '../../input-control';
+
+export const InputWrapper = styled( InputControl )`
+	${ InputControlContainer } {
+		padding: 5px 8px;
+	}
+`;
+
+export const NumberControlWrapper = styled( NumberControl )`
+	${ InputControlContainer } {
+		width: 96px;
```

Comment:
Same concern here about bespoke paddings.

On the width, what options do we have to use flex here so the items are automatically spaced and sized by the container?

Reply:
The `NumberControlWrapper` is [rendered as a child of `<Spacer as={ HStack }>`](https://github.com/WordPress/gutenberg/blob/c3a5fe294b544e536617f34867b52a9e7e219be3/packages/components/src/ui/color-picker/input-with-slider.tsx#L28), which means that `Spacer` is effective a flex container, and that [props from `HStack`](https://github.com/WordPress/gutenberg/blob/e6e0ea8906c0fb01ea132e3386d5e42172614ba9/packages/components/src/h-stack/README.md) can be applied to that instance of `<Spacer />`. `NumberControlWrapper` can be treated as a flexbox child.